front/lib/tracking/customerio/server.ts

```diff
@@ -230,6 +230,7 @@ export class CustomerioServerSideTracking {
         first_name: user.firstName,
         last_name: user.lastName,
         created_at: user.createdAt,
+        id: user.sId,
```

Comment:
should this be sId to avoid confusion?

Reply:
No strong opinion, can change if you prefer. It's not super ambiguous IMO since we should never have model IDs outside of internal backend BL. But sid is more explicit